package.json

```diff
@@ -82,15 +90,21 @@
     "gulp-tsc": "^1.3.0",
     "gulp-typescript": "^3.1.6",
     "husky": "^0.13.4",
-    "jest": "^21.1.1",
-    "jest-styled-components": "^4.9.0",
     "lint-staged": "^4.0.0",
-    "merge-stream": "^1.0.1",
+    "longjohn": "^0.2.12",
+    "morgan": "^1.8.1",
     "prettier": "^1.7.4",
-    "react": "^15.5.4",
+    "raf": "^3.4.0",
     "react-addons-test-utils": "^15.5.1",
-    "react-dom": "^15.5.4",
-    "react-test-renderer": "^15.6.1",
+    "react-dom": "^16.0.0",
+    "react-hot-loader": "^1.3.1",
+    "react-test-renderer": "^16.0.0",
+    "relay2ts": "^0.2.0",
+    "ts-babel-node": "alloy/ts-babel-node#artsy",
+    "ts-node": "^3.0.2",
+    "jest": "^21.1.1",
+    "jest-styled-components": "^4.9.0",
+    "merge-stream": "^1.0.1",
```

Comment:
This is a lot of new deps; are these changes coming in from #master?

Reply:
looks like a mistake from the merge. For sure some of these aren't needed or may be duplicates